docs/references/general/rawacf.md

```diff
@@ -1,3 +1,110 @@
+| *xcf*                   | *None*   | short     | XCF flag, indicates if XCF for elevation angle estimation is available (1) or not (0)   |
+| *tfreq*                 | *kHz*    | short     | Transmitted frequency                                                                   |
+| *mxpwr*                 | *dB*     | int       | Maximum power                                                                           |
+| *lvmax*                 | *None*   | int       | Maximum noise level allowed                                                             |
+| *combf*                 | *None*   | string    | Comment buffer, usually contains the control program name or command line command used to generate the file                            |
+| *rawacf.revision.major* | *None*   | int       | Major version number of the RAWACF algorithm                                            |
+| *rawacf.revision.minor* | *None*   | int       | Minor version number of the RAWACF algorithm                                            |
+| *thr*                   | *None*   | float     | Threshold factor                                                                        |
+
+
+### Vectors 
+
+!!! Note
+    *slist* contains the range gates that obtained data points during the integration period of the beam. 
+
+| Field name  | Units    | Dimensionality | Data Type   | Description                                                                 |
+| :---------- | :-----:  | :-------:      | :---:       | :---                                                                        |
+| *ptab*      | *None*   | *[mppul]*      | short       | Pulse table, showing when pulses are transmitted as integer multiples of mpinc/smsep                                                              |
+| *ltab*      | *None*   | *[mplgs+1][2]* | short       | Lag table, showing which pulses to use to estimate each lag                 |
+| *pwr0*      | *dB*     | *[nrang]*      | float       | Lag zero power (actually SNR), estimated from voltage samples (not fitted)  |
+| *slist*     | *None*   | *[0-nrang]*    | short       | List of range gates with ACF/XCF data available |
+| *acfd*      | *None*   | *[2][mplgs][0-nrang]*    | short       | Calculated ACFs                                                             |
+| *xcfd*      | *None*   | *[2][mplgs][0-nrang]*    | char        | Calculated XCFs                                                             |
+
+## File structure
+
+RAWACF files contain typically 2 hours of data. Individual records in the RAWACF file contain the raw data for a single integration period (usually 3s or 7s, but depends on operating mode of the radar). 
+In the standard operational mode (common mode) where each beam is scanned sequentially, the beam number `bmnum` will change each integration period. However, some control programs will stay on one beam for a whole scan (called a "camping beam"). 
```

Comment:
I'm not sure this is the usual definition of a "camping beam" - at least I usually think of a camping beam as one which is revisited more than once during a scan, rather than being sampled on each record throughout the entire scan.

Reply:
Agree with @egthomas, and I think describing camping beams goes beyond the scope of documenting the file format, so we can probably just remove it. 
```suggestion
In the standard operational mode (common mode) where each beam is scanned sequentially, the beam number `bmnum` will change each integration period. 
```

Reply:
I added it in because people look at our files and question why certain beams scans are not linear so this was a tid bit of information. Since we don't have set beam scanning information I figured we say something her for now so people are not confused when they look at the files and then reference once we have better documentation. 

More information is not bad.

Reply:
We could provide a more generic description of the beam sequences, rather than focusing on just camping beams...

```suggestion
In the standard operational mode (common mode) where each beam is scanned sequentially, the beam number `bmnum` will increase or decrease by 1 after each integration period. Other control programs may involve different beam sequences, including sampling the beams in a different order, or sampling a subset of the available beams.
```

Reply:
That sounds like a good general description of what happens to `bmnum`. I do think it's good to have this extra information as it is a common question from new/external users of what's happening during some non-standard modes.

Reply:
I think my suggestion here is fine as well